### DISCUSSION TITLE
Custom URL

### DISCUSSION THREAD

**Discussion by mik-kun** [pretix/pretix#2721] [Q&A] [answered]
Hi folks. Currently our pretix installation nd organizer points at pretix.domain.com/organizer.
We want to point it directly to tickets.domain.com for our single organizer.

According to the [documentation here ](https://docs.pretix.eu/en/latest/user/organizers/domain.html) you need a CNAME pointing to the main domain pretix.domain.com and add it to your webserver (added 'ServerAlias tickets.domain.com' to the virtual host in apache).
So far it works and I can access the alias in the browser and it opens pretix default root.

However adding the "Custom domain" value in the Organizer settings admin session doesn't do anything, it still shows me the pretix default root.

Am I missing something?

Highly appreciate any help as I'm new to this.
Thank you.

**Comment by raphaelm** [comment, marked as answer]
Looking at the previous discussion, I assume your apache config now statically sets the Host header of the main domain, could that be the case?

**Reply by mik-kun** [reply]
Hi Raphaelm, attached is my apache config. I added the ServerAlias value as mentioned.
Using nginx what do I need to add/modify to make a custom url work? I will try to translate the syntax to apache.

```
<IfModule mod_ssl.c>
	<VirtualHost *:443>
        ServerAdmin admin@domain.com
        ServerName pretix.domain.com
		ServerAlias tickets.domain.com
        DocumentRoot /var/pretix/data/media/

        <Directory />
                Options FollowSymLinks
                AllowOverride None
        </Directory>

        <Directory /var/pretix/data/media/>
                Options FollowSymLinks MultiViews Indexes
                AllowOverride All
                Require all granted
        </Directory>

        SSLProxyEngine on
        ProxyHTMLEnable Off

        ProxyPass "/"  "http://localhost:8345/"
        ProxyPassReverse "/"  "http://localhost:8345/"

		ErrorLog ${APACHE_LOG_DIR}/pretix_error.log
		CustomLog ${APACHE_LOG_DIR}/pretix_access.log combined

		SSLEngine on

		SSLCertificateFile	/xxx.crt
		SSLCertificateKeyFile /xxx.key
		SSLCertificateChainFile /xxx.crt

	</VirtualHost>
</IfModule>
```

**Reply by raphaelm** [reply]
try ProxyPreserveHost

**Reply by mik-kun** [reply]
ProxyPreserveHost On works!
Thank you so much Raphael 😊

**Comment by mik-kun** [comment]
While I'm on it, is there a way to skip the "Hello! This is a self-hosted installation" root page and forward directly to /control/?
